src/libsyncengine/update_detection/file_system_observer/computefsoperationworker.h

```diff
@@ -27,6 +27,17 @@ namespace KDC {
 class ComputeFSOperationWorker : public ISyncWorker {
     public:
         ComputeFSOperationWorker(std::shared_ptr<SyncPal> syncPal, const std::string &name, const std::string &shortName);
+        /**
+         * Constructor used for testing only
```

Comment:
This could be defined in a derived class dedicated to tests.

Reply:
Same as above